core/domain/feedback_services.py

```diff
@@ -79,7 +78,12 @@ def create_thread(
         exploration_id, state_name, original_author_id, subject, text, False)
 
 
+def get_exp_id_thread_id_list(full_thread_id):
```

Comment:
I think get_exp_id_and_thread_id might be a better name, and maybe it's worth adding a docstring here explaining. When I first saw the function name I was thinking it was a list of something (messages, perhaps?) related to the exp id and thread id.